js/gameboard.js

```diff
@@ -1,110 +1,160 @@
-$(document).ready(function(){
-  $('#share').click(function(){
-    $('#media').show();
-  });
-  $('#red').click(function(){
-    $('body').attr("class","redspraycan"); 
-  setRed();
-  });
-  $('#blue').click(function(){
-    $('body').attr("class","bluespraycan"); 
-  setBlue();
-  });
-  $('#yellow').click(function(){
-    $('body').attr("class","yellowspraycan"); 
-  setYellow();
-  });
-  $('#green').click(function(){
-    $('body').attr("class","greenspraycan"); 
-  setGreen();
-  });
-  var url = window.location.href;
-console.log(url);
-var index = url.indexOf("=");
-var img = url.substring(index+1);
-console.log(img);
-  // variable for the canvas 
-var gameboard = getCanvas();
-// variable for the image url 
-var brickwall = "https://s-media-cache-ak0.pinimg.com/736x/94/aa/66/94aa6684bdc152457d6983e58fd24bd2.jpg";
-imageInput (img, gameboard);
-drawing ();
-});
-//START OF DRAWING CODE
-//returning a new image
-function getImage (){
-  return new Image();
+/**
+ * This is a function that takes a str and a sperator ( also a string ),
+ * that strips a string with any sperator found in it.
+ * @param {string} str the given a string to transform
```

Comment:
the given a string?

Reply:
the given string you want to transform

ex:
```js
stripString('hey', 'y')
```

Reply:
Just edit the comment a bit to make it a little easier to read.